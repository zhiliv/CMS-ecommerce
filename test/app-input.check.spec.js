import { shallowMount } from '@vue/test-utils'
import H from '@/components/app/input-check/input-check.vue'

describe('app-input-check.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(H, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        classes: 'simple simple-xl',
        styles: 'background: white',
        id: 'indent',
        value: 'тест',
        placeholder: 'Предпоказ текста',
        type: 'radio'
      },
      slots:{
        default: 'Содержимое'
      }
    })
  })
  it('Содержимое html для app-input-check ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
