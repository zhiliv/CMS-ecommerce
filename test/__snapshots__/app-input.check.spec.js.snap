// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-input-check.js Содержимое html для app-input-check  1`] = `
<input
  class="simple simple-xl"
  id="indent"
  placeholder="Предпоказ текста"
  style="background: white;"
  type="radio"
  value="тест"
/>
`;
